contrib/golang/filters/http/source/go/pkg/api/filter.go

```diff
@@ -69,6 +69,8 @@ type StreamInfo interface {
 	AttemptCount() uint32
 	// Get the dynamic metadata of the request
 	DynamicMetadata() DynamicMetadata
+	// DownstreamLocalAddress return the downstream local address.
```

Comment:
does this API return the client address, or the server(envoy) address? I'm confused about it.
see: https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/lua_filter#downstreamlocaladdress
> Returns the string representation of [downstream remote address](https://github.com/envoyproxy/envoy/blob/ea341eba3152aeda0ef0579bca61f5a1fe483203/envoy/stream_info/stream_info.h) used by the current request.

Reply:
it is the the server(envoy) address, maybe the lua doc is error, i will submit another pr to fix, thanks.